README.md

```diff
@@ -86,7 +86,7 @@ _We are using `composer.json` file for `tests` folder, so that you will have to
 
 ### Run tests
 
-To run the tests please execute the following commands. We are usin [Robo.li](http://robo.li/) to execute the tests.
+To run the tests please execute the following commands. We are using [Robo.li](http://robo.li/) to execute the tests.
```

Comment:
Please also mention codeception somewhere :-) And that also the phpunit tests are run.